webpack.config.js

```diff
@@ -16,6 +16,11 @@ const config = {
         test: /\.jsx?/,
         include: APP_DIR,
         loader: 'babel-loader'
+      },
+      {
+        test: /\.css?/,
```

Comment:
awesome, thanks for that @rckchen

Reply:
Welcome! However this isn't a working product as we also need style-loader as well 
https://github.com/webpack-contrib/style-loader